interfaces/auth_token_validator.yaml

```diff
@@ -13,5 +13,10 @@ cmds:
         Result object containing validation result
       type: object
       $ref: /authorization#/ValidationResult
+vars:
+  validate_result_update:
+    description: Updated validation result for a given token due to some changes.
```

Comment:
You could add in the description that it was added because OCPP can do an update in `StartTransaction.req` or `TransactionEvent.conf` . It makes it a little easier to argue about it later if we need to